### PR TITLE
Create SummationDecapode.json

### PR DESCRIPTION
**Pull request by jpfairbanks**
@cthoyt,
  This is the JSON file from #32. 

-JPF

### CODE REVIEW

File: src/acsets/schemas/catlab/SummationDecapode.json [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            "dom": "Var"
+        },
+        {
+            "name": "name",
+            "codom": "Name",
+            "dom": "Var"
+        }
+    ]
+}
+{
```
Comment (cthoyt) [inline src/acsets/schemas/catlab/SummationDecapode.json:108]: is the content duplicated?

File: src/acsets/schemas/catlab/SummationDecapode.json [hunk 2]
```diff
@@ -0,0 +1,213 @@
+
+{
+    "version": {
+        "ACSets": "0.0.0",
```
Comment (cthoyt) [inline src/acsets/schemas/catlab/SummationDecapode.json:4]:
```suggestion
        "Catlab": "0.14.12",
```

File: src/acsets/schemas/catlab/SummationDecapode.json [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        },
+        {
+            "name": "sum",
+            "codom": "Var",
+            "dom": "Σ"
+        }
+    ],
+    "AttrType": [
+        {
+            "name": "Type"
```
Comment (cthoyt) [inline src/acsets/schemas/catlab/SummationDecapode.json:76]: all AttrType need a `ty` field with a datatype

File: src/acsets/schemas/catlab/SummationDecapode.json [hunk 4]
```diff
@@ -0,0 +1,213 @@
+
+{
+    "version": {
+        "ACSets": "0.0.0",
+        "ACSetSchema": "0.0.1"
+    },
+    "Ob": [
+        {
+            "name": "Var"
```
Comment (cthoyt) [inline src/acsets/schemas/catlab/SummationDecapode.json:9]: Ideally, all `Ob`s have a "title" that has a human readable name. The schema should provide some kind of information for anyone who is getting data in this shape what the data is

File: src/acsets/schemas/catlab/SummationDecapode.json (hunk 2 above)
Comment (jpfairbanks) [reply src/acsets/schemas/catlab/SummationDecapode.json:3]: I actually generated this with ACSets v0.0.0 so I think this is right, but is this version supposed to be a minimum compatible Catlab version?

Comment (cthoyt) [reply src/acsets/schemas/catlab/SummationDecapode.json:3]: The Catlab schema in this repo requires it to be catlab... not really sure I can comment on why it's this way or not. Good question for @olynch

**Comment by jpfairbanks** [comment]
Do these titles look good?

**Comment by cthoyt** [comment]
> Do these titles look good?

Yes! You can see I have pushed the converter script and the results.

**Review by cthoyt — APPROVED** [review]
@jpfairbanks this is ready for merge. We can double back later to fill in more metadata (ideally upstream)

**Pull request merged**